providers/instance.rb

```diff
@@ -0,0 +1,28 @@
+use_inline_resources
+
+def whyrun_supported?
+  true
+end
+
+action :create do
+  config_dir = node['ganeti-instance-image']['config_dir']
+  directory "#{config_dir}/networks/instances/" do
```

Comment:
@ramereth Great, just wanted to ask first to make sure I wasn't stepping on any toes.